### PR TITLE
feat: Create EVM to Substrate example

### PR DESCRIPTION
**Pull request by enemycnt**
## Related Issue Or Context

Closes: #179 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: examples/evm-to-substrate-react-example/src/components/Form.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+
+import { useEvm } from "../evm-lib";
+// import { evmSetupList } from "../config";
```
Comment (wainola) [inline examples/evm-to-substrate-react-example/src/components/Form.tsx:5]: remove commented code

File: examples/evm-to-substrate-react-example/src/components/Form.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        <label htmlFor="to" className="label">
+          Destination EVM network:
+        </label>
+        <input
+          type="text"
+          placeholder="domainId"
+          {...register("to")}
+          className="input"
+        />
+        {/* <select {...register("to")} className="input">
```
Comment (wainola) [inline examples/evm-to-substrate-react-example/src/components/Form.tsx:73]: remove commented code

Comment (enemycnt) [reply examples/evm-to-substrate-react-example/src/components/Form.tsx]: removed

File: examples/evm-to-substrate-react-example/src/components/Form.tsx (hunk 1 above)
Comment (enemycnt) [reply examples/evm-to-substrate-react-example/src/components/Form.tsx]: removed

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**